app/tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Lets switch to cdn hosted version and dark mode by default as you suggested earlier, having two distinct 280k line files really sucks